Migrations.cs

```diff
@@ -36,16 +36,16 @@ public Migrations(IContentDefinitionManager contentDefinitionManager)
 
         #region Migrations
 
-        public int Create()
+        public int CreateAsync()
         {
-            UpdateFrom1();
+            UpdateFrom1Async();
 
             return 2;
         }
 
-        public int UpdateFrom1()
+        public int UpdateFrom1Async()

```

Comment:
This needs to change so it returns a `Task<int>` and is marked as async as shown below.

```
public async Task<int> UpdateFrom1Async() 
{
    ...
}
```